src/behavior-considered-undefined.md

```diff
@@ -23,30 +23,49 @@ code.
 </div>
 
 * Data races.
-* Dereferencing a null or dangling raw pointer.
-* Unaligned pointer reading and writing outside of [`read_unaligned`]
-  and [`write_unaligned`].
-* Reads of [undef] \(uninitialized) memory.
-* Breaking the [pointer aliasing rules] on accesses through raw pointers;
-  a subset of the rules used by C.
-* `&mut T` and `&T` follow LLVM’s scoped [noalias] model, except if the `&T`
-  contains an [`UnsafeCell<U>`].
-* Mutating non-mutable data &mdash; that is, data reached through a shared
-  reference or data owned by a `let` binding), unless that data is contained
+* Dereferencing (using the `*` operator on) a dangling or unaligned raw pointer.
+* Breaking the [pointer aliasing rules]. `&mut T` and `&T` follow LLVM’s scoped
+  [noalias] model, except if the `&T` contains an [`UnsafeCell<U>`].
+* Mutating non-mutable data (that is, data reached through a shared
+  reference or data owned by an immutable binding), unless that data is contained
   within an [`UnsafeCell<U>`].
```

Comment:
Considering [this discussion](https://github.com/rust-lang/rust/pull/63955#discussion_r318447052), I think I'll have to add a clause like "this is always UB for memory in a `const`, no matter the `UnsafeCell`".

Reply:
Yes please!